Sprint 2/Main.java

```diff
@@ -0,0 +1,87 @@
+package me.logan.senpagamemory;
+
+import java.io.IOException;
+

```

Comment:
Overall, the code looks well-formatted, it's easy to read, and the design looks robust overall.